webapp-django/crashstats/topcrashers/views.py

```diff
@@ -104,8 +105,19 @@ def get_topcrashers_results(**kwargs):
                 if row['term'] in (1, -1):
                     hit['hang_count'] += row['count']
 
-            # Number of startup crashes.
-            hit['startup_percent'] = 0
+            # Number of crashes happening during startup. This is defined by
+            # the software, as opposed to the next methid which relies on
+            # the uptime of the software.
+            hit['startup_count'] = 0
+
+            sig_startup = hit['facets']['startup_crash']
+            for row in sig_startup:
+                if row['term'] in ('T', '1'):
+                    hit['startup_count'] = row['count']
```

Comment:
Yeah, the fact that it can be either ``T`` or ``0`` is temporary, that's due to the mapping having changed. In the future all values will be either ``T`` or ``F`` or missing. But that ``sum`` looks better for now.